jekyll/_cci2/creating-orbs.md

```diff
@@ -0,0 +1,332 @@
+
+Namespaces are owned by organizations. Only organization administrators can create namespaces.
+
+Organizations are, by default, limited to claiming only one namespace. This policy is designed to limit name-squatting and namespace noise. If you require more than one namespace please contact your account team at CircleCI.
+
+#### Development vs. Production Orbs
+
+Orb versions may be added to the registry either as development versions or production versions. Production versions are always a semantic version like 1.5.3; whereas development versions can be tagged with a string and are always prefixed with dev:.
+
+##### Development and Production Orb Security Profiles
+
+* Only organization administrators can publish production orbs.
+
+* Any member of an organization can publish dev orbs in namespaces.
+
+##### Development and Production Orb Retention and Mutability Characteristics
+
+* Dev orbs are mutable and expire. Anyone can overwrite any development orb who is a member of the organization that owns the namespace in which that orb is published.
+
+* Production orbs are immutable and long-lived. Once you publish a production orb at a given semantic version you may not change the content of that orb at that version. To change the content of a production orb you must publish a new version. CircleCI generally recommends using the ```orb publish increment``` and/or the ```orb publish promote``` commands in the ```circleci``` CLI when publishing orbs to production.
+
+##### Development and Production Orbs Versioning Semantics
+
+Development orbs are tagged with the format ```dev:<< your-string >>```. Production orbs are always published using the semantic versioning ("semver") scheme.
+
+In development orbs, the string label given by the user has the following restrictions:
+
+* Up to 1023 non-whitespace characters
+
+Examples of valid development orb tags:
+
+* Valid:
+  "dev:mybranch"
+  "dev:2018_09_01"
+  "dev:1.2.3-rc1"
+  "dev:myinitials/mybranch"
+  "dev:myVERYIMPORTANTbranch"
+
+* Invalid
+  "dev: 1" (No spaces allowed)
+  "1.2.3-rc1" (No leading "dev:")
+
+In production orbs you must use the form ```X.Y.Z``` where ```X``` is a "major" version, ```Y``` is a "minor" version, and ```Z``` is a "patch" version. For example, 2.4.0 implies the major version 2, minor version 4, and the patch version of 0.
+
+While not strictly enforced, CircleCI recommends when versioning your production orbs you use the standard semantic versioning convention for deciding what major, minor, and patch should mean semantically:
+
+* MAJOR: when you make incompatible API changes
+* MINOR: when you add functionality in a backwards-compatible manner
+* PATCH: when you make backwards-compatible bug fixes
+
+#### Using Orbs in Orbs and Register-Time Resolution
+
+You may use an ```orbs``` stanza inside an orb, providing a way to pull in other orbs' elements into your orb. The scoping rules are the same as orbs in configuration, and you declare inline orbs in an orb.
```

Comment:
This could use a little more word-smithing, but it succeeds in reducing the use of the word `orb` from seven times in two sentences to four times in two sentences, so I'm calling it a small win.

> Orbs can reference other orbs in their source code. The scoping rules are the same as in `.circleci/config.yml` configuration: use an ```orbs``` stanza to declare your orb dependencies.

Reply:
I just simplified this to You may use an ```orbs``` stanza inside an orb.